Riot/SupportingFiles/Info.plist

```diff
@@ -6,21 +6,6 @@
 	<string>en</string>
 	<key>CFBundleDisplayName</key>
 	<string>$(BUNDLE_DISPLAY_NAME)</string>
-	<key>CFBundleDocumentTypes</key>
-	<array>
-		<dict>
-			<key>CFBundleTypeName</key>
-			<string>Mention Pills</string>
-			<key>CFBundleTypeRole</key>
-			<string>Viewer</string>
-			<key>LSHandlerRank</key>
-			<string>Owner</string>
-			<key>LSItemContentTypes</key>
-			<array>
-				<string>im.vector.app.pills</string>
-			</array>
-		</dict>
-	</array>
```

Comment:
I don't think we should revert the changes for mention pills?

Reply:
I've simplified the changes

Reply:
I believe Xcode sometimes reformats Plists when editing because those lines appear to just have been moved around, not actually changed.

Reply:
How did I not notice this 🙈